ui.apps/src/main/content/jcr_root/apps/etoolbox-link-inspector/clientlibs/link-inspector-ui/js/console-ui.delete.js

```diff
@@ -0,0 +1,79 @@
+/*
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+/**
+ * EToolbox Link Inspector clientlib.
+ * "Filter" action definition.
+ */
+(function (window, document, $, ELC, Granite, Coral) {
+    'use strict';
+    const DIALOG_TITLE_LABEL = Granite.I18n.get('Delete Report');
+    const CANCEL_LABEL = Granite.I18n.get('Cancel');
+    const SUBMIT_FILTER_LABEL = Granite.I18n.get('Delete');
+
+    function onFilterAction(name, el, config, collection, selections) {
+        const dialog = document.querySelector('#delete--dialog');
```

Comment:
(cosmetic) Why "--" and not "-" ?) It kind of appears like BEM but "dialog" is not "modifier"